tree/master/code/string_algorithms/sum_of_numbers_string/sum_of_numbers_string.py

```diff
@@ -0,0 +1,20 @@
+# ADDING ALL NUMBERS IN A STRING.
+
+CODE:
```

Comment:
Fix all the comments and remove this line.

Reply:
Made changes. Please review.